### DISCUSSION TITLE
Requesting ARM 64 support, spatial extension file currently not hosted on extensions.duckdb.org

### DISCUSSION THREAD

**Discussion by nikhil-scholarlify** [duckdb/duckdb-spatial#300] [Ideas]
Hi, I'd posted at https://github.com/duckdb/duckdb/discussions/11591 and was sent here.

I'm on an Arm64 CPU architecture server which appears as an alternative to x86 on most server platforms these days. This one's in Azure. I'm not able to load duckdb spatial extension there. Sharing minimum reproducible code:

```
import duckdb
cursor = duckdb.connect()
print(cursor.execute('INSTALL spatial; LOAD spatial;').fetchall())
```
This fails with error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
duckdb.duckdb.HTTPException: HTTP Error: Failed to download extension "spatial" at URL "http://extensions.duckdb.org/v0.9.2/linux_arm64_gcc4/spatial.duckdb_extension.gz"
Extension "spatial" is an existing extension.
```

So the url http://extensions.duckdb.org/v0.9.2/linux_arm64_gcc4/spatial.duckdb_extension.gz is invalid and returns 403 error, I presume such a file isn't hosted on extensions.duckdb.org site. 

On the other hand, http://extensions.duckdb.org/v0.9.2/linux_amd64_gcc4/spatial.duckdb_extension.gz (with a-m-d instead of a-r-m) is present on the site and gets fetched when I run this code in an x86 / amd64 cpu-architecture machine.

An aside : I'm originally trying to run quackosm and in addition to spatial, that one is also looking for a parquet extension file with same results as above.

So, please build and host the arm64 extension files on extensions.duckdb.org, or please show how I can build them myself and configure duckdb to pick those up instead of trying an invalid url.

Addendum : I see that duckdb latest version is v0.10.1 whereas my code here is showing v.0.9.2 : that's because I'm using quackosm originally and in there the duckdb version is fixed to v0.9.2 . In any case, the url http://extensions.duckdb.org/v0.10.1/linux_arm64_gcc4/spatial.duckdb_extension.gz if I bump the version number there, is also giving 403 error.

**Comment by RaczeQ** [comment]
Hi, I'm the developer of the QuackOSM library. Starting from version [0.6.1](https://github.com/kraina-ai/quackosm/releases/tag/0.6.1) it has the support for the 3.12 Python version and arm64 after upgrading to duckdb 0.10.2 version.

If you have any issues using QuackOSM - please add them to the main repository 🤗 https://github.com/kraina-ai/quackosm (I also saw https://github.com/duckdb/duckdb/discussions/11591 related to quackosm as well)
